lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -291,6 +312,7 @@ def create_network(self):
                 resp = self.client.create_network(self.parameters.network_name,
                                                   driver=self.parameters.driver,
                                                   options=self.parameters.driver_options,
+                                                  scope=self.parameters.scope,
```

Comment:
Note that `scope` requires docker-py 2.6.0 or later ([changelog](https://docker-py.readthedocs.io/en/stable/change-log.html)). Since docker_network also supports docker-py 1.8.0, you need to only add this argument if create_network supports it. Search for `HAS_CPUSET_MEMS_OPT` in docker_container to see how this can be done.